pkgs/servers/rundeck/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  name = "rundeck-${version}";
+  version = "2.6.2";
+
+  src = fetchurl {
+    url = "http://dl.bintray.com/rundeck/rundeck-maven/rundeck-launcher-${version}.jar";
+    sha256 = "1vcj00zwkjqk62b7ikm538s5y08w7y68l0czx8pa2yv0v9s3wx42";
+  };
+  meta = with stdenv.lib; {
+    description = "Job scheduler and runbook automation";
+    homepage = http://rundeck.org;
+    license = licenses.asl20;
+    platforms = platforms.all;
+    maintainers = [ ];
+  };
+
+  buildCommand = "ln -s $src $out";
```

Comment:
As `rundeck` is just one jar, I'm wondering if it makes sense to make a derivation because it can't be used without running it manually with `java -jar`.

Using `fetchurl` in the service module would also be an option. Opinions?